### ISSUE TITLE
mlr3extralearners::create_learner should work on tempfile()

### ISSUE THREAD

**Issue by jakob-r** [mlr-org/mlr3extralearners#52]
It would be nice if `create_learner()` could also write the learner in a tempfile() or somewhere else than just the package in case you want to only use if for yourself.

**Comment by RaphaelS1** [comment]
Makes sense, can just add a `file` argument that specifies location to save learner to and if NULL adds to package. 

Would you also want the tests to be removed?

**Comment by RaphaelS1** [comment]
@jakob-r I  just noticed this is still open, I can make a PR for this now but let me know what you think about testing

**Comment by sebffischer** [comment]
this is implemented (0.5.43)
